stdlib/public/core/StringUTF8.swift

```diff
@@ -240,20 +259,30 @@ extension String {
       internal static var _bufferHiByte: Buffer {
         return 0xFF << numericCast((sizeof(Buffer.self) &- 1) &* 8)
       }
+      
+      /// Amounts to increment the UTF-16 index based on the high nibble of a
+      /// UTF-8 code unit. If the high nibble is:
+      ///
+      /// - 0b0000-0b0111: increment 1
+      /// - 0b1000-0b1011: continuation character, increment 0
+      /// - 0b1100-0b1110: increment 1
+      /// - 0b1111: UTF-16 surrogate pair, increment 2
+      internal static var _u16Increments = Int(bitPattern:
```

Comment:
Please use a computed var instead.  Static variables are lazily-initialized, and accesses to them can include synchronization.